<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TSF_Model_Client</class>
 <widget class="QMainWindow" name="TSF_Model_Client">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1004</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>381</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="data_graph_view">
       <property name="title">
        <string>Data Graph View</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>571</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="data_table_view">
       <property name="title">
        <string>Data Table View</string>
       </property>
       <widget class="QLabel" name="input_now">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>67</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Input now</string>
        </property>
       </widget>
       <widget class="QLabel" name="predict_table_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>67</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Predict</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="input_tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>551</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="predict_tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>551</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>381</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="Control">
       <property name="title">
        <string>Control</string>
       </property>
       <widget class="QPushButton" name="select">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>80</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="path_edit">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Input data file path</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Path_btn">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>40</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Path</string>
        </property>
       </widget>
       <widget class="QLabel" name="input_data_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ContextMenuPolicy::PreventContextMenu</enum>
        </property>
        <property name="text">
         <string>Input Data</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="predict">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>120</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Predict</string>
        </property>
       </widget>
       <widget class="QLabel" name="forecast">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Forecast</string>
        </property>
       </widget>
       <widget class="QPushButton" name="show_graph">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>120</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Graph</string>
        </property>
       </widget>
       <widget class="QLabel" name="edit_path_print">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="text">
         <string>file path</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
